src/app/contact/container/Contact2.tsx

```diff
@@ -1,10 +1,65 @@
 import React from 'react'
-import Image from 'next/image'
 
 export default function Contact2() {
   return (
-    <div className="bg-blue-500">
-      <h1>안녕안녕!!</h1>
+    <div className="overflow-x-hidden">
+      <div className="desktop:px-80 ml-16 gap-16 tablet:px-80 ml-16 gap-16 mobile:px-16 ml-4 gap-32 flex flex-col ">
```

Comment:
링크 목록들이 반복되는 형태니까,
조금 더 시맨틱하게 마크업한다면은
`<ul> - <li>` 패턴으로 마크업할 수 있을 거 같습니다!
```suggestion
      <ul className="desktop:px-80 ml-16 gap-16 tablet:px-80 ml-16 gap-16 mobile:px-16 ml-4 gap-32 flex flex-col ">
```

Reply:
ul 요소에 바로 직접적인 자식 요소들을 li로 마크업해주시면 될 거 같습니다:)